### ISSUE TITLE
Please continue this project

### ISSUE THREAD

**Issue by Cyber688** [wwylele/MApRX#28]
See the Kirby hacks that are out there right now I think that there is potential.
There is a rom hack named Kirby Super Star Allies that has great level design but it's story couldn't be implemented due to limitations.
So please make it so we can edit the dialogue in the game.

**Comment by wwylele** [comment]
Oh wow, I didn't expect an issue opened to my super old project. At this point this repo is so foreign to me...

Dialogue editing is going to be a completely different subject from what his project intended to do. I'd need to start from scratch to see what I can do about it

**Comment by Wyattmo** [comment]
Yeah there is actually a small community working on rom hacks and making discoveries about the game. If you continued the project in any shape or form it would be amazing. If you would like to check it out here is the discord https://discord.gg/SPNMWNHzzz

**Comment by Cyber688** [comment]
@wwylele Please remake MApRX and make it easy to use.
